### DISCUSSION TITLE
Find out record that cause TransactionException

### DISCUSSION THREAD

**Discussion by yoonghm** [duckdb/duckdb#7061] [Q&A]
Hi,

I am using Python client to access duckdb.  I have a unique index on 3 columns of a table. When I inserted data into the table, I encountered this:

```
TransactionException: TransactionContext Error: Failed to commit: Constraint Error: PRIMARY KEY or UNIQUE constraint violated: duplicated key
```

Are there sample code to catch the exception and display the affected record?

**Comment by Tishj** [comment]
I'm fixing this in #7050 👍
